src/test/java/com/javabom/lotto/domain/ticket/LottoTicketDispenserTest.java

```diff
@@ -0,0 +1,51 @@
+package com.javabom.lotto.domain.ticket;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoTicketDispenserTest {
+
+    @DisplayName("로또 발급기가 원하는대로 로또 티켓을 발급하는지 확인")
+    @Test
+    void getAutoTickets() {
+        // given
+        LottoTicketDispenser lottoTicketDispenser = new LottoTicketDispenser(new FixedLottoNumberGenerator());
+        List<LottoNumber> stubLottoNumbers = IntStream.rangeClosed(1, 6)
+                .boxed()
+                .map(LottoNumber::new)
+                .collect(Collectors.toList());
+        LottoTicket stubLottoTicket = new LottoTicket(stubLottoNumbers);
+        LottoTickets expectedLottoTickets = new LottoTickets(Arrays.asList(stubLottoTicket));
+
+        // when
+        Money money = new Money(1_000);
+        LottoTickets receivedLottoTicket = lottoTicketDispenser.getAutoTickets(money);
+
+        // then
+        assertThat(receivedLottoTicket).isEqualTo(expectedLottoTickets);
+    }
+
+    @DisplayName("로또 발급기가 원하는 개수의 티켓을 뽑아내는지 확인")
+    @Test
+    void name() {
+        // given
+        LottoTicketDispenser lottoTicketDispenser = new LottoTicketDispenser(new RandomLottoNumberGenerator());
+
+        // when
+        int expectedTicketQuantity = 1;
+        Money money = new Money(1_000 * expectedTicketQuantity);
+        LottoTickets lottoTickets = lottoTicketDispenser.getAutoTickets(money);
+
+        // then
+        assertThat(lottoTickets.get().size()).isEqualTo(expectedTicketQuantity);
```

Comment:
👍LottoTickets가 의도한 대로 생성하는지가 테스트의 목적이니까 그 내부에서 사용되는 객체가 실제 프로덕션 코드에서 사용되는 거 일수록 의미가 있다고 생각해

그리고 이렇게 바꿀수 있어
```suggestion
        assertThat(lottoTickets.get()).hasSize(expectedTicketQuantity);
```